_includes/meta.html

```diff
@@ -24,13 +24,17 @@
   <meta property="og:title" content="{{title}} - Data.gov" />
   <meta property="og:description" content="{{excerpt}}" />
   <meta property="og:site_name" content="{{ site.title }}"/>
+  {% assign img = "https://s3-us-gov-west-1.amazonaws.com/cg-0817d6e3-93c4-4de8-8b32-da6919464e61/hero-image-bg.png" %}
```

Comment:
When we have a new image in the future, this img will have a broken link. can we add a test on this to ensure the link is valid?

Reply:
Yes, that is a good idea. Ideally when/if we update the image we just replace the old image with a new with the same filename. I do think it would be wise to make sure that it is a valid image, though. Working on a test now.

Reply:
can you test it with this shortcode on your local and see if that works?

```
// Image exists in S3
{% image "my-image.png" "My PNG Image Alternative Name" false %}
```

https://github.com/GSA/datagov-11ty/tree/main?tab=readme-ov-file#referencing-images

Reply:
or better, we can make it a 11ty config item, so that it will always point to the latest url if the image is updated in the 11ty config. 

I dont like relying on developer to remember to edit this line whenever there is a image update.
I dont like adding a test either. Even there is a updated image, the old image might not get deleted from S3 therefore the old url is still valid and will pass the test.

Reply:
> or better, we can make it a 11ty config item, so that it will always point to the latest url if the image is updated in the 11ty config.

wouldn't we then have to update the config whenever the image is updated? 🌀

Reply:
> Working on a test now.

Did you ever get that test finished @robert-bryson ?